HomeWork 4/HomeWork 4/Program.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
+
+class Program
+{
+    static void Main()
+    {
+        Console.Write("N = ");
+        int N = Convert.ToInt32(Console.ReadLine());
+        int[] arr = new int[N];
+        Console.WriteLine($"Enter {N} elements:");
+        for (int i = 0; i < N; i++)
+        {
+            Console.Write($"Element {i + 1}:");
+            arr[i] = Convert.ToInt32(Console.ReadLine());
+        }
+        Console.WriteLine();
+        Console.Write("Array elements: ");
+        foreach (int element in arr)
+        {
+            Console.Write(element + "; ");
+        }
+
+        Random rnd = new Random();
+
+        for (int i = 0; i < N; i++)
+        {
+            arr[i] = rnd.Next(1, 27);
+        }
+        Console.WriteLine();
+        Console.Write("Array elements: ");
+        foreach (int element in arr)
+        {
+            Console.Write(element + "; ");
+        }
+        Console.WriteLine();
+
+        Console.WriteLine("Odd numbers from 1 to 26: ");
+        for (int i = 1; i <= 26; i++)
+        {
+            if (i % 2 != 0)
+            {
+                Console.Write(i + "; ");
+            }
+        }
+        Console.WriteLine();
+
+        Console.WriteLine("Even numbers from 1 to 26: ");
+        for (int i = 1; i <= 26; i++)
+        {
+            if (i % 2 == 0)
+            {
+                Console.Write(i + "; ");
+            }
+        }
```

Comment:
These parts looks like similar, we can move this lodic to a separate method and reuse it